src/menus/table.tsx

```diff
@@ -1,16 +1,63 @@
-import { TrashIcon } from "outline-icons";
+import {   
+  TrashIcon,
+  InsertLeftIcon,
+  InsertRightIcon,
+  InsertAboveIcon, 
+  InsertBelowIcon,
+} from "outline-icons";
 import { MenuItem } from "../types";
 import baseDictionary from "../dictionary";
 
+
+interface TableMenuItemsInput {
+  rowIndex: number;
+  rtl: boolean;
+}
+
 export default function tableMenuItems(
-  dictionary: typeof baseDictionary
+  dictionary: typeof baseDictionary,
+  tableIndexes: TableMenuItemsInput,
 ): MenuItem[] {
+  const { rowIndex, rtl } = tableIndexes;
+
   return [
     {
       name: "deleteTable",
       tooltip: dictionary.deleteTable,
       icon: TrashIcon,
       active: () => false,
     },
+    {
+      name: "separator",
+    },
+    {
+      name: "addRowBefore",
+      tooltip: dictionary.addRowBefore,
+      icon: InsertAboveIcon,
+      attrs: { index: ( rowIndex as number ) - 1 },
+      active: () => false,
+    },
+    {
+      name: "addRowAfter",
+      tooltip: dictionary.addRowAfter,
+      icon: InsertBelowIcon,
+      attrs: { index: ( rowIndex as number ) },
+      active: () => false,
+    },
+    {
+      name: "separator",
+    },
+    {
+      name: rtl ? "addColumnAfter" : "addColumnBefore",
+      tooltip: rtl ? dictionary.addColumnAfter : dictionary.addColumnBefore,
+      icon: InsertLeftIcon,
+      active: () => false,
```

Comment:
have you tested it with `rtl` direction? just to make sure it's working 😄

Reply:
i just tested and it looks like the whole table formatting gets messed up :-( let me tweek

Reply:
im just gonna put a ticket to fix RTL in editor cause it looks like other parts of the editor is messed up with rtl as well